applications/mne_analyze/mne_analyze/mainwindow.cpp

```diff
@@ -390,10 +398,13 @@ void MainWindow::createPluginControls(QSharedPointer<ANSHAREDLIB::PluginManager>
     setTabPosition(Qt::RightDockWidgetArea,QTabWidget::East);
     setDockOptions(QMainWindow::ForceTabbedDocks);
 
+    pPluginManager->setParent(this);
```

Comment:
PluginManager is part of AnalyzeCore where it is held as a QSharedPointer. Setting the parent here might interfere with QSahredPointer's automated garbage collection. Do we need to set the parent here?